src/Param/Param.cpp

```diff
@@ -41,25 +62,69 @@ void Param::set_name(const std::string name)
     {
         throw NOMAD::Exception("Param.cpp", __LINE__, "Param name is not valid");
     }
-    _name = name;
+    m_name = name;
 }
 
-ParamValue Param::get_value() const
+ParamValue Param::get_paramvalue() const
 {
-    return _value;
+    return m_paramvalue;
```

Comment:
See there is confusion between and param value. Should be cleared. User of the lib does not have to know about the ParamValue class.